test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -193,7 +203,100 @@ def _run_tag_success(image_uri, ec2_client, ec2_instance, ec2_connection):
 
     preexisting_ec2_instance_tags = ec2_utils.get_ec2_instance_tags(ec2_instance_id, ec2_client=ec2_client)
     if expected_tag_key in preexisting_ec2_instance_tags:
-        ec2_client.remove_tags(Resources=[ec2_instance_id], Tags=[{"Key": expected_tag_key}])
+        ec2_client.delete_tags(Resources=[ec2_instance_id], Tags=[{"Key": expected_tag_key}])
+
+    if "tensorflow" in framework and job_type == "inference":
```

Comment:
Did this piece come from anywhere or did we never properly test tf inference?

Reply:
Its a copy paste code from other func in same file https://github.com/aws/deep-learning-containers/pull/2239/files/32a48d8273dcdecc286f38326003c1fafb075a15#diff-a80f3063ba4837df91fd3c0137d220e46a795f7805f76ba6ac8806d428312f44R158

Reply:
let me try to reuse it by defining it in func